core/dbt/contracts/graph/unparsed.py

```diff
@@ -87,6 +87,11 @@ class Docs(dbtClassMixin, Replaceable):
     node_color: Optional[str] = None
 
 
+@dataclass
+class Contract(dbtClassMixin, Replaceable):
```

Comment:
It doesn't look like the Contract class is used by anything in unparsed.py, so it ought to go in nodes.py.  We should limit what nodes.py uses from unparsed.py as much as possible.

Reply:
That creates circular imports with model_config.  Instead of `nodes.py` I moved it into `model_config.py`.  The only other thing defined in there is `Hook` but I think it still makes sense since it's not used in `unparsed.py`.